docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,196 @@
+* Setting up your e-mail address [reference](https://help.github.com/articles/setting-your-commit-email-address-in-git/)
+
+Once Git and your GitHub account have been properly configured, you can add the -S flag to the git commits:
+
+```console
+$ git commit -S -m your commit message
+# Creates a signed commit
+```
+
+### Commit message format
+
+This repository uses the guidelines for commit messages outlined in [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/)
+
+## Pull Requests
+
+If this is your first time contributing to an open-source project on GitHub, make sure you read about [Creating a pull request](https://help.github.com/en/articles/creating-a-pull-request).
+
+A pull request must always link to at least one GitHub issue. If that is not the case, create a GitHub issue and link it.
+
+To increase the chance of having your pull request accepted, make sure your pull request follows these guidelines:
+
+* Title and description matches the implementation.
+* Commits within the pull request follow the formatting guidelines.
+* The pull request closes one related issue.
+* The pull request contains necessary tests that verify the intended behavior.
+* If your pull request has conflicts, rebase your branch onto the main branch.
+
+If the pull request fixes a bug:
+
+* The pull request description must include `Fixes #<issue number>`.
+* To avoid regressions, the pull request should include tests that replicate the fixed bug.
+
+The owning team _squashes_ all commits into one when we accept a pull request. The title of the pull request becomes the subject line of the squashed commit message. We still encourage contributors to write informative commit messages, as they become a part of the Git commit body.
+
+We use the pull request title when we generate change logs for releases. As such, we strive to make the title as informative as possible.
+
+Make sure that the title for your pull request uses the same format as the subject line in the commit message.
+
+### Quality Gates for pull requests
+
+GitHub Actions are used to enforce quality gates when a pull request is created or when any commit is made to the pull request. These GitHub Actions enforce our minimum code quality requirement for any code that gets checked into the code repository. If any of the quality gates fail, it is expected that the contributor will look into the check log, understand the problem and resolve the issue. If help is needed, please feel free to reach out to the maintainers of the project for [support](SUPPORT.md).
+
+#### Code build/test/coverage
+
+[GitHub action](https://github.com/dell/<<Action link>>) that runs unit tests and checks that the code coverage of each package meets a configured threshold (currently 90%). An error is flagged if a given pull request does not meet the test coverage threshold and blocks the pull request from being merged.
```

Comment:
We have updated the section as "[GitHub action](https://github.com/ansible-collections/dellemc.enterprise_sonic/actions) that runs unit tests automatically and checks that the code coverage of each package meets a configured threshold (currently 90%). An error is flagged if a given pull request does not meet the test coverage threshold and blocks the pull request from being merged. When it fails, it is expected that the contributor will look into the log, understand the problem and resolve the issue."

Reply:
Updated as suggested.

Reply:
This is a better description, but I still have a couple of minor comments. Since I have posted them as a comment in the file (with a specific request for changed content), I will mark this issue "Resolved" to simplify tracking of the remaining issues.